scripts/cli-test.sh

```diff
@@ -47,6 +48,7 @@ INVOICE_STATUS="$(echo $INVOICE_RESULT | jq -r '.status')"
 
 # incoming lightning
 INVOICE="$($FM_MINT_CLIENT ln-invoice '100000msat' 'integration test' | jq -r '.ln_invoice.invoice')"
+await_cln_block_processing
```

Comment:
Don't think we mined blocked since the last call, but doesn't hurt to have it.

Reply:
Ahh yea, I'll remove